iocore/net/SSLClientUtils.cc

```diff
@@ -66,6 +66,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
   netvc = SSLNetVCAccess(ssl);
 
   if (netvc != nullptr) {
+    netvc->callHooks(TS_EVENT_SSL_SERVER_VERIFY_HOOK);
```

Comment:
Regarding use case for bypassing/overriding OpenSSL validation:
- A plugin that fetches missing intermediate CAs via AIA information
- MitM plugin that wants to verify some upstream certs according to user based policy
- CRL cache plugin